content/asset-tracking/using-the-sdks.textile

```diff
@@ -115,96 +126,105 @@ val publisher = Publisher.publishers() // get the Publisher builder in default s
         }))
 ```
 
-You can read more about Token Authentication in our "documentation":/core-features/authentication#ably-jwt-process.
+```[javascript]
+// authURL is the endpoint for your authentication server. It returns a JWT
+const subscriber = new Subscriber({ 
+  authUrl: 'http://my.website/auth',
+  clientId: 'CLIENT_ID'
+})
+```
 
-h2(#publishing-sdk). Using the publishing SDK
+You can read more about JWT authentication in our "documentation":/core-features/authentication#ably-jwt-process.
 
-Common operations you need to carry out on the publisher include:
+blang[android].
+  h2(#publishing-sdk). Using the publishing SDK
```

Comment:
"Subscribing SDK" is capitalised so "publishing SDK" should be too.

Reply:
Yes it should! Fixed.